src/print-markdown.ts

```diff
@@ -201,24 +184,13 @@ No files were changed`
   ${modulesBySizeDescending
     .map(chunkModule => printChunkModuleRow(chunkModule))
     .join('\n')}`
-  const detailedTable = `${DETAILED_CHUNK_TABLE_HEADERS}
-  ${modulesBySizeDescending
-    .map(chunkModule => printDetailedChunkModuleRow(chunkModule))
-    .join('\n')}`
 
   return `
 <details>
 <summary>**Changeset**</summary>
```

Comment:
Minor thing: can we fix this header to remove the asterisks? It was supposed to be bold, but that doesn't work 😅

Reply:
next time - I had automerge on for this